.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report 🐞
+about: Report a bug found in the YAML Language Server
+title: ''
+labels: 'kind/bug'
+assignees: ''
+
+---
+
+## Describe the bug
+<!--- Provide a summary of the issue.
```

Comment:
I think we should ask for an example of their yaml or schema. That is almost always the first question asked on incoming bugs.